### DISCUSSION TITLE
Specifying different video file path on remote machine

### DISCUSSION THREAD

**Discussion by tomhagley6** [talmolab/sleap#1120] [Help!]
Hi, I'm trying to label on one machine and train my model on a different one. This means I need to specify a different video path during training, but I don't seem to be able to. The template job profiles don't have an option for this, and I've tried:

trainer = sleap.nn.training.TopDownMultiClassModelTrainer.from_config(
    cfg, training_labels=labels, video_search_paths=videos

But still get the same FileNotFoundError: Could not find filename

Any help would be appreciated. Thanks!

**Comment by roomrys** [comment]
Hi @tomhagley6,

Well, I think we have found ourselves a bit of a logical error in SLEAP. I traced where we use `video_search_paths`  when creating a `Trainer` via `from_config` and ended up in our `DataReaders.from_labels` method where I discovered that the `video_search_paths` are only used if the `training_labels` argument is a path to an slp (not the already loaded `Labels` object):
https://github.com/talmolab/sleap/blob/833c2d5bdcf4cfef4e0adc7569b8e2245494a8fa/sleap/nn/training.py#L155-L171
and while we update the field in the training config for `search_path_hints` (if the `Labels` object is already loaded), 
https://github.com/talmolab/sleap/blob/833c2d5bdcf4cfef4e0adc7569b8e2245494a8fa/sleap/nn/training.py#L132-L142
I cannot seem to find where we actually use this **pretty useful** `search_path_hints` field. I am converting this discussion to an issue to get this resolved. 

In the meantime, could you try passing in the path to the slp to `training_labels` instead of the `Labels` object and let us know how that works.

Thanks,
Liezl
